src/utils/ao.ts

```diff
@@ -120,3 +121,35 @@ export async function evolveANT({
 
   return id;
 }
+
+export function createAoSigner(
+  signer: ContractSigner,
+): (args: {
+  data: string | Buffer;
+  tags?: { name: string; value: string }[];
+  target?: string;
+  anchor?: string;
+}) => Promise<{ id: string; raw: ArrayBuffer }> {
```

Comment:
update the type here now that we have `AoSigner`

Reply:
https://github.com/ar-io/ar-io-sdk/pull/175/commits/8e95eddf5723b0fdbf870688556f76a3e80bb6fa